service/worker/deletenamespace/workflow.go

```diff
@@ -67,6 +67,7 @@ var (
 	localRetryPolicy = &temporal.RetryPolicy{
 		InitialInterval: 1 * time.Second,
 		MaximumInterval: 10 * time.Second,
+		MaximumAttempts: 3,
```

Comment:
Should we add Maximum attempts here, as the API client will wait for this activity to finish before it gets the response?
Should we make MaximumAttempts 1, so that API client will not have to wait a few seconds for the result?

Reply:
Everything which is executed as local activity should be lightweight, quick, and shouldn't fail. In case if database is down, I would prefer this activity to retry up to WF timeout rather than return an error.

Reply:
Got it. But we are not setting any timeout for DeleteNamespaceWorkflow. Should I add a timeout?